Octokit/Models/Response/Deployment.cs

```diff
@@ -80,6 +89,21 @@ public Deployment(int id, string nodeId, string sha, string url, User creator, I
         /// </summary>
         public string StatusesUrl { get; protected set; }
 
+        /// <summary>
+        /// The API URL for the <seealso cref="Repository"/>es of this deployment.
+        /// </summary>
+        public string RepositoryUrl { get; protected set; }
+
+        /// <summary>
+        /// The name of the <seealso cref="Environment"/> that was deployed to (e.g., staging or production).
+        /// </summary>
+        public string @Environment { get; protected set; }
+
+        /// <summary>
+        /// 
```

Comment:
I know there's no description on the github documentation but how about something like:
The name of the <seealso cref="Environment"/> that was originally deployed to (e.g., staging or production).

Reply:
done